deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -372,7 +372,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
             var cell = capacitySetting.currentCell;
             var scope = this;
             var provider = capacitySetting.currentProvider;
-            value = 'cmp_base-ebs-18.04';
+            value = '{{ default_cmp_image }}';
```

Comment:
can we just use black as default type and throw error form is submitted with black?

Reply:
It's used in the basic form creation. We should get rid of basic form in the future, but have to keep it default until those design considerations can be done.

Reply:
I think using " " as default should be fairly straightforward to do now but leave it up to you.

Reply:
It requires moving the empty field to the basic form. I think it's better in more of a design discussion rather than adding a new field to the basic form in this PR.